packages/psss/src/components/DropdownMenu.js

```diff
@@ -153,4 +156,5 @@ DropdownMenu.propTypes = {
     }),
   ).isRequired,
   onChange: PropTypes.func.isRequired,
+  readOnly: PropTypes.bool,
```

Comment:
Let's add a default prop for that (I have a feeling that you linter is disabled 😃 )